src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/VideoBooking.kt

```diff
@@ -45,7 +45,7 @@ class VideoBooking private constructor(
 
   var probationMeetingType: String?,
 
-  @Deprecated(message = "This is to be superseded by notesForStaff and notesForPrisoners.")
+  @Deprecated(message = "This is superseded by notesForStaff and notesForPrisoners and not longer used.")
   var comments: String? = null,
```

Comment:
Left on JPA entity for now, could probably be removed (but left on the DB table).

Reply:
Done